src/qt/forms/optionsdialog.ui

```diff
@@ -206,6 +206,16 @@
           <string>Expert</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
+          <item>
+           <widget class="QCheckBox" name="subFeeFromAmount">
+            <property name="toolTip">
+             <string>Whether to Subtact Fee From Amount or not.</string>
+            </property>
+            <property name="text">
+             <string>Subtract &amp;Fee From Amount</string>
```

Comment:
For any string in a UI file, you can add an `extracomment` field within the opening string tag. The value of this field is equivalent to a translator comment in our qt code

```suggestion
             <string extracomment="An options menu setting to always subtract the fee from the amount a user is sending.">Subtract &amp;Fee From Amount</string>
```